src/Svg.css

```diff
@@ -0,0 +1,9 @@
+.svgWrapper {
+    display: inline-block;
+    vertical-align: center;
+
+    & svg {
+        display: block;  /* Remove little margin between the svg and its parent */
```

Comment:
That sentence is half-true. 
SVG by default is a display: inline element, so it takes into consideration the line-height (and you probably have an empty character before or after the element), that's why it changes the height.

I wouldn't call it a margin, I would just remove the comment 🤷